### ISSUE TITLE
Add examples to amp-image-slider.md

### ISSUE THREAD

**Issue by nainar** [ampproject/amphtml#18058]
### First Timers Only
We know figuring out the process for contributing to an open source project can be intimidating, so we created this issue as a way for you to learn the ropes.  (If you feel comfortable contributing to open source projects, please leave this issue for someone else.)

### What you will need to know
How to edit markdown files

### Background

The amp-image-slider.md file doesn't have examples on how to use the different attributes. Please add one example each of how to use the different attributes:
- `disable-hint-reappear`
- `initial-slider-position`
- `step-size`

### Step by step
- [ ] Claim this issue by adding a comment below.  Please only claim this bug if you plan on starting work in the next day or so.  (If you [join the AMP Project](https://goo.gl/forms/T65peVtfQfEoDWeD3) we'll be able to assign this issue to you after you've claimed it.)
- [ ] If you aren't too familiar with Git/GitHub, see the [Getting Started End-to-End Guide](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md) for [an intro to Git & GitHub,](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#intro-to-git-and-github) and [how to get a copy of the code](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#get-a-copy-of-the-amphtml-code).  You can also refer to the [Quick Start Guide](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-quick.md) for the necessary setup steps with less explanation than the End-to-End guide.
- [ ] Follow the instructions for [building AMP](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#building-amp-and-starting-a-local-server).
- [ ] [Create a Git branch](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#create-a-git-branch) for making your changes.
- [ ] [Sign the Contributor License Agreement](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#contributor-license-agreement) before creating a Pull Request.  (If you are contributing code on behalf of a corporation start this process as early as possible.)

- [ ] [Commit your changes](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#edit-files-and-commit-them) frequently.
- [ ] [Push your changes to GitHub](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#push-your-changes-to-your-github-fork).

- [ ] [Create a Pull Request](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#send-a-pull-request-ie-request-a-code-review).  Mention `closes Issue <this issue number>` in the description.
- [ ] [Respond to your reviewer's comments](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#respond-to-pull-request-comments) (if any).

Once approved, your changes will be merged.  **⚡⚡⚡Congrats on making your first contribution to the AMP Project!⚡⚡⚡**  You'll be able to see it [live across the web soon](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md)!

Thanks, and we hope to see more contributions from you soon.

### Questions?

If you have questions ask in this issue or on your Pull Request (if you've created one) or see the [How to get help](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#how-to-get-help) section of the Getting Started guide.

**Comment by CrystalOnScript** [comment]
@nainar @Enriqe  - I see there's been a related PR, can this issue be closed?

**Comment by Enriqe** [comment]
I think my branch got messed up at the time and pushed that commit by accident. Other than that I don't have context on this, sorry!

**Comment by shntm** [comment]
@nainar was looking to contribute for the first time, but I see that [the examples are already added](https://github.com/ampproject/amphtml/commit/a068e4d537886563f69781816456c1c7c3fa4b85).
Is there anything left to do in this issue?

**Comment by caroqliu** [comment]
Fixed by #18590
